src/test/groovy/graphql/nadel/RemovedFieldsTest.groovy

```diff
@@ -42,6 +43,36 @@ class RemovedFieldsTest extends StrategyTestHelper {
         result.errors[0].message.contains("removed field")
     }
 
+    def "all hydrated fields in query are removed"() {
```

Comment:
I am not sure about these tests: I am missing assertions for the queries we send to the underlying service.
I think these tests should be more like  "underlying service returns null for non-nullable field" in NadelExecutionStrategyTest2.

Reply:
Is that better?
`    def "underlying service returns null for non-nullable field"() {
        given:
        def overallSchema = TestUtil.schemaFromNdsl('''
        service Issues {
            type Query {
                issue: Issue
            }
            type Issue {
                id: ID!
            }
        }
        ''')
        def issueSchema = TestUtil.schema("""
            type Query {
                issue: Issue
            }
            type Issue {
                id: ID!
            }
        """)
        def query = "{issue {id}}"

        def expectedQuery1 = "query nadel_2_Issues {issue {empty_selection_set_typename__UUID:__typename}}"
        def response1 = [issue: [empty_selection_set_typename__UUID: "Query"]]

        Map overallResponse
        List<GraphQLError> errors
        when:
        (overallResponse, errors) = test1Service(
                overallSchema,
                "Issues",
                issueSchema,
                query,
                ["issue"],
                expectedQuery1,
                response1,
                createServiceExecutionHooksWithFieldRemoval(["id"])
        )
        then:
        overallResponse == [issue: [id: null]]
        errors.size() == 1
        errors[0].message == "removed field"
    }
`